recipes/flann/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    string: {{PKG_BUILDNUM}}_g{{GIT_FULL_HASH[:7]}}
```

Comment:
I am guessing this is because we are using a development version here. Could you please give us some clarification as to why the development version is needed and why this one in particular?